website/src/app/views/about/members.js

```diff
@@ -300,7 +306,6 @@ const members = {
       name: 'Donghyun Daniel Koo',
       position: 'Developer',
       linkedinURL: 'https://www.linkedin.com/in/koo-donghyun/',
-      imageURL: daniel,
```

Comment:
Is there a reason for deleting this line?

Reply:
Since the image was being used twice, I kept getting an error that said that the identifier was already called. So it turns out that they were accidentally using my photo, but I think they meant to use their own image.

Reply:
It looks like you replaced "Donghyun Daniel Koo"'s image (website/src/images/about/memberImages/daniel.jpg) with your image. So that is why it shows the same image for both of you guys. The way to fix this is to add "Donghyun Daniel Koo"'s image back and to rename your image to something other than just "Daniel" (e.g. you can name your image "danielreyes") DM me on slack if you need help :)